server/api/section/section.controller.js

```diff
@@ -61,6 +61,58 @@ function removeEntity(res) {
   };
 }
 
+function updateSection(req){
+  if(req.body.pendingStudent){
+    return updatePendingStudents(req);
+  }
+  else{
+    return saveSectionUpdates(req);
+  }
```

Comment:
:penguin: I feel like this should probably be a separate endpoint- if i try to submit pending students the rest of my request won't work? Seems like weird behavior no?